### DISCUSSION TITLE
Load multiples images

### DISCUSSION THREAD

**Discussion by Gfernandezv** [JingyiGF/HERBS#52] [Q&A]
First of all, thank you very much for your code, it's great. I have a question, is it possible to add multiple images in the same project? If so, how is it done?

greetings

**Comment by JingyiGF** [comment]
Hi, I think due to memory(RAM) usage limitations HERBS does not allow users to open multiple histological images at once. But uses are allowed to open another image when they finished processing the previous one. So you do not have save projects for every single image, instead you may process images sequentially and then save the project. But in this way, the path to the image will be the last one being processed. What do you want to do? process multiple images? Or save multiple images? For processing, ok. saving, no.
